include/SDL3/SDL_gpu.h

```diff
@@ -1769,16 +1795,16 @@ extern SDL_DECLSPEC void SDLCALL SDL_GpuCopyTextureToTexture(
  * \param copyPass a copy pass handle
  * \param source the buffer to copy from
```

Comment:
Maybe `the buffer and offset to copy from`? Same deal with the line below

Reply:
Done!